packages/ui/src/kit/VAccordion/VAccordionItem.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div
+    :tabindex="0"
+    :class="itemClassName"
+
+    @click="handleClick"
```

Comment:
onClick

Reply:
@dudiq you use `nandleClick` in many places, there are no conventions or nobody respect these conventions.

Reply:
@ifiri
if you need func only in template subs we use `on` prefix
if you need call func in methods we use `handle` prefix or better use normal method name such `createSome, removeSome` or other without `handle` prefix